### PR TITLE
add kspace properties and theorems

### PR DESCRIPTION
**Pull request by StevenClontz**
See #230

### CODE REVIEW

File: properties/P000140.md [hunk 1]
```diff
@@ -7,7 +7,12 @@ aliases:
...
+  - mathse: 4646084
+    name: Unraveling the various definitions of k-space or compactly generated space
 ---
 
-Subsets are closed (resp. open) whenever their intersection with every compact
+Subsets are closed (resp. open) exactly when their intersection with every compact
 subset is closed (resp. open) in the subspace topology.
+
+Equivalently, subsets are closed (resp. open) exactly when their intersection with every continuous
+image of a compact space is closed (resp. open) in the subspace topology.
```
Comment (prabau) [inline properties/P000140.md:18]: Shouldn't this be instead that a subset $U\subseteq X$ is open in $X$ if for every continuous map $f:K\to X$ with $K$ an arbitrary compact space, the inverse image $f^{-1}(U)$ is compact in $K$?  And same thing for P141.  Maybe we could mention "final topology" also?

File: properties/P000141.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+uid: P000141
+slug: compactly-generated-t2-maps
+name: Generated by compact Hausdorff maps
+aliases:
+    - Compactly generated
+    - k-space
+    - Kelley space
```
Comment (prabau) [inline properties/P000141.md:8]: Where does Kelley space come from?  As far as I know, the notion of compactly generated in Kelley's book of 1955 was "coherent with the collection of closed compact subspaces", which is different.

Comment (StevenClontz) [reply properties/P000141.md:8]: Took this from https://ncatlab.org/nlab/show/compactly+generated+topological+space (also uses "kaonic space")

File: properties/P000140.md (hunk 1 above)
Comment (StevenClontz) [reply properties/P000140.md:18]: "image $f^{-1}(U)$ is *open* in K" I think, yes. I was being lazy when copy-pasting definitions and didn't think about this carefully.

I think these are equivalent when $f$ is an inclusion map from a subspace; it's less clear that these are equivalent when $f$ is not injective (and I suspect they aren't).

**Comment by StevenClontz** [comment]
@prabau feel free to edit on this branch directly, or I'll return to this later this week.

File: properties/P000141.md (hunk 2 above)
Comment (prabau) [reply properties/P000141.md:8]: Checking the pdf linked from the nlab article, Kelley space is defined as Hausdorff space with P140.  I don't think we should even mention this in pi-base.

**Comment by prabau** [comment]
I am wondering what the best terminology for these spaces is.  "Generated" is ok, and so is maybe "determined" (as introduced by Gruenhage et al, https://projecteuclid.org/journals/pacific-journal-of-mathematics/volume-113/issue-2/Spaces-determined-by-point-countable-covers/pjm/1102709194.full).  Not sure what my preference is, just writing it down to make it easier to decide.

Some possibilities for P142: 
- Generated by compact Hausdorff subspaces
- Determined by compact Hausdorff subspaces

For P141:
- Generated by compact Hausdorff maps (maybe not so clear what a compact map is?)
- Generated by maps from compact Hausdorff spaces (clear, but too long?)
- Generated by maps from compact Hausdorff
- Determined by ...

**Comment by StevenClontz** [comment]
Is "determined" correct for P141? It's defined in terms of the subspace topology within f[K], not the inverse image within K.

**Comment by prabau** [comment]
> Is "determined" correct for P141? It's defined in terms of the subspace topology within f[K], not the inverse image within K.

Yeah, it does not really apply there.

**Review by ccaruvana — APPROVED** [review]

**Pull request merged**